lib/OpenQA/Test/PatchDeparse.pm

```diff
@@ -15,20 +15,21 @@ if (
   )
 {
 
-#<<<  do not let perltidy touch this
+#<<<  do not let perltidy nor perlcritic touch this
+## no critic (TestingAndDebugging::ProhibitNoStrict)
 # This is not our code, and formatting should stay the same for
 # better comparison with new versions of B::Deparse
 # <---- PATCH
 package B::Deparse;
 no warnings 'redefine';
 no strict 'refs';
 
-*{"B::Deparse::walk_lineseq"} = sub {
+*{'B::Deparse::walk_lineseq'} = sub {
```

Comment:
The comment above says "This is not our code, and formatting should stay the same for better comparison with new versions of B::Deparse" so changing the formatting here seems wrong.

Reply:
The quote type changes are irrelevant to the inner workings of the code. It'll perform *exactly* the same task regardless of the quote.

Why keeping it out-of-style?

Reply:
"for better comparison with new versions of B::Deparse" - although I'm just quoting the comment here. I guess I personally don't care much about it.

Reply:
for context: https://progress.opensuse.org/issues/40895

Reply:
@josegomezr Be sure to resolve comments, or otherwise respond if you consider the matter addressed. Otherwise it looks like there's an open question here.

Reply:
Yes, tabs to spaces is a change.

Reply:
just to confirm: tabs to spaces is not acceptable for this file in particular?

Reply:
That part of the code should stay exactly the same. Only in this file, for the reason stated in the comment.